test/problems/test_lasso_small_strongly_convex.jl

```diff
@@ -0,0 +1,47 @@
+using LinearAlgebra
+using Test
+using Random
+
+using ProximalOperators
+using ProximalAlgorithms
+
+@testset "Lasso small (strongly convex, $T)" for T in [Float32, Float64]
+
+    Random.seed!(777)
+    dim = 5
+    μf = T(1.0)
+    Lf = T(10.0)
+
+    x_star = convert(Vector{T}, 1.5 * rand(T, dim) .- 0.5)
+
+    lam = convert(T, (μf + Lf) / 2.0)
+    @test typeof(lam) == T
+
+    D = Diagonal(convert(Vector{T}, sqrt(μf) .+ (sqrt(Lf) - sqrt(μf)) * rand(T, dim)))
+    D[1] = sqrt(μf)
```

Comment:
Why all these `sqrt`? Shouldn't the diagonal elements of `D` just be between `mu` and `L` to get the desired strong convexity and smoothness?

Reply:
Based on the current code, the eigenvalues of `A` are between `sqrt(mu)` and `sqrt(L)`. Moreover, the Hessian of `f` is `A' * A`. Hence, we have the desired condition

- strong convexity of `f` = minimum eigenvalue of `A' * A` = (minimum eigenvalue of `A`) ^ 2 =  `μf`
-  smoothness of `f` = maximum eigenvalue of `A' * A` = (maximum eigenvalue of `A`) ^ 2 = `Lf`

Reply:
Of course, I overlooked it 🙂